client/src/actions/dataset.js

```diff
@@ -58,6 +58,46 @@ export function fetchDataset(id, metaOnly) {
   };
 }
 
+function fetchSortedDatasetRequest(id) {
+  return {
+    type: constants.FETCH_SORTED_DATASET_REQUEST,
```

Comment:
Could we call it (and the functions related) fetch_coincidences_sorted_dataset ?